ResourceManager/GitExtensionsControl.cs

```diff
@@ -74,6 +74,18 @@ public virtual void TranslateItems(ITranslation translation)
 
         #region Hotkeys
 
+        /// <summary>Checks if a hotkey wants to handle the key and execute it.</summary>
```

Comment:
Can you describe the relation to `ProcessCmdKey` a little here?
I get the overview but cannot suggest something...

Reply:
done